src/main/java/org/opensearch/neuralsearch/processor/InferenceProcessor.java

```diff
@@ -62,6 +65,7 @@ public abstract class InferenceProcessor extends AbstractProcessor {
 
     private final Environment environment;
     private final ClusterService clusterService;
+    List<Pair<String, Object>> mappingForNestedFields;
```

Comment:
Can we make it as private? why should it be package scope?

Reply:
not relevant as I'm reverting back to previous approach